More TR edits (#1343)
hadley/r4ds: data-tidy.qmd

### Many variables in column names

A more challenging situation occurs when you have multiple pieces of information crammed into the column names, and you would like to store these in separate new variables.
For example, take the `who2` dataset, the source of `table1` and friends that you saw above:

```{r}
who2
```

> This dataset collected by the WHO records information about tuberculosis diagnoses.

Reviewer: ```suggestion
This dataset, collected by the World Health Organisation, records information about tuberculosis diagnoses.
```

There are two columns that are already variables and are easy to interpret: `country` and `year`.
They are followed by 56 columns like `sp_m_014`, `ep_m_4554`, and `rel_m_3544`.
If you stare at these columns for long enough, you'll notice there's a pattern.
Each column name is made up of three pieces separated by `_`.

> The first piece, `sp`/`rel`/`ep`, describes the method used for the diagnosis, the second piece, `m`/`f` is the `gender` (which is coded as a binary variable in this dataset), and the third piece, `014`/`1524`/`2535`/`3544`/`4554`/`65` is the `age` range (`014` represents 0-14, for example).

Reviewer: ```suggestion
The first piece, `sp`/`rel`/`ep`, describes the method used for the diagnosis, the second piece, `m`/`f` is the `gender` (coded as a binary variable in this dataset), and the third piece, `014`/`1524`/`2535`/`3544`/`4554`/`65` is the `age` range (`014` represents 0-14, for example).
```

cms_patient_experience
```

> The core unit being studies is an organization, but each organization is spread across six rows, with one row for each measurement taken in the survey organization.

Reviewer: ```suggestion
The core unit being studied is an organization, but each organization is spread across six rows, with one row for each measurement taken in the survey organization.
```
